sentry-compose/src/androidMain/res/drawable/sentry_user_feedback_compose_button_logo_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android" android:height="24dp" android:viewportHeight="24" android:viewportWidth="24" android:width="24dp">
```

Comment:
It's probably safer to add some attribution, [e.g. like we did here.](https://github.com/getsentry/sentry-java/blob/8b8369f06cbc5a9738de7810b1df5863b3ac6bcb/sentry-android-core/src/main/java/io/sentry/android/core/ANRWatchDog.java#L1-L24)

Reply:
Hmm good point, it's pretty unclear which license this is then. But maybe add a link to the source? Seems to be [https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:android/resources/images/material/icons/materialicons/campaign/baseline_campaign_24.xml](https://cs.android.com/android-studio/platform/tools/adt/idea/+/mirror-goog-studio-main:android/resources/images/material/icons/materialicons/campaign/baseline_campaign_24.xml)